phoenix-core/src/main/java/org/apache/phoenix/execute/HashJoinPlan.java

```diff
@@ -411,7 +431,12 @@ public void postProcess(Object result, HashJoinPlan parent)
                     + " IN (" + keyRangeRhsExpression.toString() + ")";
             return Collections.<String> singletonList(step);
         }
-        
+
+
+        @Override
+        public QueryPlan getInnerPlan() {
+            return plan;
+        }
     }
 }
 
```

Comment:
Looks like we're maxing out GitHub -- no more diffs below here. Maybe if the other refactors it would be better?